packages/evm/contracts/test/arbitration/Arbitrable.sol

```diff
@@ -4,7 +4,7 @@ import "../../arbitration/IArbitrable.sol";
 import "../../arbitration/IArbitrator.sol";
 
 
-contract ArbitrableMock is IArbitrable {
+contract Arbitrable is IArbitrable {
```

Comment:
Actually I like the "Mock" name, since this is really a dummy version of an Arbitrable that we wouldn't want anyone to use :)

Reply:
Addressed in 3dc65487976aaacb8864552bab8211e7cccffcc7